pennylane/measurements/measurements.py

```
@@ -396,15 +399,19 @@ def expand(self):
         None
         """
         if self.obs is None:
-            raise qml.operation.DecompositionUndefinedError
+            raise DecompositionUndefinedError
 
-        with qml.queuing.AnnotatedQueue() as q:
+        with AnnotatedQueue() as q:
             self.obs.diagonalizing_gates()
             self.__class__(wires=self.obs.wires, eigvals=self.obs.eigvals())
 
-        return qml.tape.QuantumScript.from_queue(q)
+        from pennylane.tape import (  # tach-ignore pylint: disable=import-outside-toplevel
```

Why do we need to tach ignore here?

---

measurements *shouldn't* depend on tapes.

---

If you want you can add `"pennylane.tape"` to the `cannot_depend_on` list for this module. That way this will actually suppress an error.